src/js/components/organisms/index.js

```diff
@@ -1,23 +1,23 @@
 import ArticleContent from 'organisms/article-content/ArticleContent';
 import ArticleHeader from 'organisms/article-header/ArticleHeader';
 import Brands from 'organisms/brands/Brands';
-export { default as Colors } from 'organisms/colors';
-export { default as ContentCover } from 'organisms/content-cover';
-export { default as ContentGeneral } from 'organisms/content-general';
-export { default as ContentHighlight } from 'organisms/content-highlight';
-export { default as HeroArticle } from 'organisms/hero-article';
-export { default as HeroBasic } from 'organisms/hero-basic';
-export { default as HeroCover } from 'organisms/hero-cover';
-export { default as HeroHomepage } from 'organisms/hero-homepage';
-export { default as HeroProduct } from 'organisms/hero-product';
+import Colors from 'organisms/colors/Colors';
+import ContentCover from 'organisms/content-cover/ContentCover';
+import ContentGeneral from 'organisms/content-general/ContentGeneral';
+import ContentHighlight from 'organisms/content-highlight/ContentHighlight';
+import HeroArticle from 'organisms/hero-article/HeroArticle';
+import HeroBasic from 'organisms/hero-basic/HeroBasic';
+import HeroCover from 'organisms/hero-cover/HeroCover';
+import HeroHomepage from 'organisms/hero-homepage/HeroHomepage';
+import HeroProduct from 'organisms/hero-product/HeroProduct';
+import JobBoardFrame from 'organisms/job-board-frame/JobBoardFrame';
+import Overview from 'organisms/overview/Overview';
+import OverviewUbr from 'organisms/overview-ubr/OverviewUbr';
+import PageFooter from 'organisms/page-footer/PageFooter';
+import PageHeader from 'organisms/page-header/PageHeader';
+import SearchResults from 'organisms/search-results/SearchResults';
+import SubNavigation from 'organisms/subnavigation/SubNavigation';
+import TabbedContent from 'organisms/tabbed-content/TabbedContent';
+import TabbedTable from 'organisms/tabbed-table/TabbedTable';
 export { default as LocaleSelector } from 'organisms/locale-selector';
```

Comment:
Why are this component and the component on line 23 the only ones that are still using the `export default as component from XYZ` instead of `import component from XYZ`?

Reply:
Those haven't been converted yet.